package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "start": "node index.js",
     "dev": "nodemon --ignore *.test.js index.js",
-    "test": "jest"
+    "prettier": "prettier --write 'index.js' 'src/*.js'",
+    "lint": "prettier --list-different 'index.js' 'src/*.js'",
+    "test": "npm run lint --silent && jest"
```

Comment:
I also added the "lint" script (renamed since it's not only used on CI) here. Worth noting, I used `npm run lint` instead of `yarn lint` since everyone has `npm` installed, but not everyone has `yarn` installed.